City-of-Pori/pori-kada
# PORI-55: Project liftupts styles

Project liftup styles.

To test:
1. drush cc all and fra
2. promote some liftups with project flags.
3. go to frontpage and confirm 2 liftups with project flags are promoted and look good.
